### PR TITLE
Tong/362 faq

### PR DESCRIPTION
**Pull request by supertong**
Demo:

image

### CODE REVIEW

File: src/app/components/FAQ/FAQ.tsx [hunk 1]
```diff
@@ -12,8 +12,8 @@ export const FAQ: React.FC<FAQProps> = () => {
 
   return (
     <div className="container mx-auto flex flex-col gap-2 p-6">
-      <h3 className="mb-4 font-bold">FAQ</h3>
-      <div className="flex flex-col gap-4">
+      <h3 className="mb-8 font-normal text-4xl text-primary">FAQ’s</h3>
```
Comment (totraev) [inline src/app/components/FAQ/FAQ.tsx:15]: You can use `core-ui` here
```suggestion
      <Heading as="h3" variant="h4" className="mb-8">FAQ’s</Heading>
```

**Review by jeremy-babylonlabs — APPROVED** [review]
lgtm!

**Review by totraev — CHANGES_REQUESTED** [review]

**Comment by totraev** [comment]
Hey @supertong for some reason styles from `bbn-core-ui` were not added. I've pushed a fix, so you need go through your changes again and update styles

File: src/app/components/FAQ/Section.tsx [hunk 2]
```diff
@@ -1,37 +1,41 @@
...
-          <motion.p
-            initial={{ opacity: 0 }}
-            animate={{ opacity: 1 }}
-            exit={{ opacity: 0 }}
-            className="mt-4"
-            dangerouslySetInnerHTML={{ __html: content }}
-          />
-        )}
-      </AnimatePresence>
+    <div className="cursor-pointer border-primary-light/20 pt-6 first:pt-0 pb-2 first:pb-0">
```
Comment (totraev) [inline src/app/components/FAQ/Section.tsx:17]:
```suggestion
    <div className="border-primary-light/20 pt-6 first:pt-0 pb-2 first:pb-0">
```

**Review by totraev — CHANGES_REQUESTED** [review]

File: src/app/components/FAQ/data/questions.ts [hunk 3]
```diff
@@ -13,28 +13,28 @@ export const questions = (
...
     },
     {
       title: "How does Bitcoin Staking work?",
-      content: `<p>${coinName} holders lock their ${coinName} using the trustless and self-custodial Bitcoin Staking script for a predetermined time (timelock) in exchange for voting power in an underlying PoS protocol. In return, Bitcoin holders will earn PoS staking rewards.</p><br />
-        <p>Finality providers perform the voting. A ${coinName} staker can create a finality provider by itself and self-delegate or delegate its voting power to a third-party finality provider.</p><br />
-        <p>If a finality provider attacks the PoS system, the ${coinName}s behind the voting powers delegated to it will be subject to protocol slashing. This deters ${coinName} stakers and finality providers from attacking the PoS system.</p>
+      content: `${coinName} holders lock their ${coinName} using the trustless and self-custodial Bitcoin Staking script for a predetermined time (timelock) in exchange for voting power in an underlying PoS protocol. In return, Bitcoin holders will earn PoS staking rewards.<br />
```
Comment (gbarkhatov) [inline src/app/components/FAQ/data/questions.ts:20]: <br /> is not parsed https://i.imgur.com/qOR9vs3.png

**Review by gbarkhatov — APPROVED** [review]

**Review by totraev — APPROVED** [review]

**Pull request merged**